database/migrations/2020_12_12_203153_migrate_admin_user.php

```diff
@@ -18,15 +18,13 @@ class MigrateAdminUser extends Migration
 	 */
 	public function up(): void
 	{
-		$user = new User();
+		$user = User::query()->findOrNew(0);
+		$user->incrementing = false; // disable auto-generation of ID
+		$user->id = 0;
+		Configs::dropCache();
```

Comment:
Why do we need to clean the cache here? How should the cash become invalid?

In particular, this migration is form the year 2020. Why do we suddenly need this change?

Reply:
See https://github.com/LycheeOrg/Lychee/pull/1491#discussion_r956577674 .